ONSdigital/census-rm-acceptance-tests-2021
# passes flag to sample_loader to return sample_units, ups version of s…

# Motivation and Context
load_sample_file now has option to store and return the loaded sample units

# What has changed
Upped version of sample_loader to v1.0.1, added flag to load_sample_file

# How to test?
With the sample_loader from this ticket
You will have to change the the pipfile in the ATs to 
census-rm-sample-loader = {editable = true,git = "https://github.com/ONSdigital/census-rm-sample-loader", ref = "load-sample-flag-to-return-sample-units”} 

Run the tests and they should continue to work

# Links
https://trello.com/c/luQsADkR/640-enhance-sampleloader-script-8
